lib/scihist_digicoll/task_helpers/subject_creator_adjuster.rb

```diff
@@ -0,0 +1,40 @@
+module ScihistDigicoll
+  module TaskHelpers
+    class SubjectCreatorAdjuster
```

Comment:
can you add some comments saying what this thing does? Maybe mention the rake task.

Reply:
mentioni t only updates subject and creator, etc.

Reply:
Yeah of course